api/lib/domain/read-models/JuryCertificationSummary.js

```diff
@@ -23,6 +34,7 @@ class JuryCertificationSummary {
       this.status = STARTED;
     }
     this.pixScore = pixScore;
+    this.hasAcquiredCleaCertification = partnerCertificationStatusToDisplayName[hasAcquiredCleaCertification];
```

Comment:
**suggestion (non bloquant)**
le `this.hasAcquiredCleaCertification` n'est pas un booléen ici non, c'est directement le status ?
```suggestion
    this.cleaCertificationStatut = partnerCertificationStatusToDisplayName[hasAcquiredCleaCertification];
```